changelog/UNRELEASED.md

```diff
@@ -1,12 +1,28 @@
-
-### New Features
-
-- None (thanks nobody)
-
 ### Bug Fixes
 
-- None (thanks nobody)
+- Fix image expandos not appearing on multireddit modqueue pages (thanks @larsjohnsen)
+- Fix gif/video expandos duplicating when expanded after an error (thanks @larsjohnsen)
+- Fix nightmode styling for private subreddit landing page (thanks @thybag)
+- Fix comments getting selected too early, causing link numbers `[1]` to shift expando icons (thanks @larsjohnsen)
+- Fix floatingSidebar always creating a scroll bar (thanks @erikdesjardins)
+- Advanced video controls are now completely hidden when showVideoControls is disabled (thanks @larsjohnsen)
+- Fix Never Ending Reddit autoLoad option not always being acknowledged (thanks @larsjohnsen)
+- Fix tabs not appearing on /r/Dashboard (thanks @larsjohnsen)
+- Fix custom filtering by subreddit when visiting the subreddit (thanks @andytuba)
+- Fix Dashboard widget configuration overflowing and getting cut off (thanks @erikdesjardins)
+- Fix Thing-based macro placeholders in the root-level comment box (thanks @andytuba)
+- Rework Wikipedia expandos (thanks @thybag)
+- Fix mediaBrowseMode not working when moving down automatically after hiding a post (thanks @larsjohnsen)
+- Fix Dashboard not saving in Edge (thanks @erikdesjardins)
+- Fix `Cannot read property 'children' of null` for some Chrome users (thanks @erikdesjardins)
+- Fix followLinkAndCommentsNewTab keyboard shortcut not working depending on the configuration of Single Click Opener (thanks @erikdesjardins)
 
 ### Housekeeping / Other
 
-- None (thanks nobody)
+- Add GitHub issue template (thanks @andytuba)
+- Tidier setup of commentsLinkNumbers keyboard shortcuts (thanks @larsjohnsen)
+- Add an option (requireDirectLink) to only display the subreddit info popup on direct links to subreddits, enabled by default (thanks @erikdesjardins)
+- Never Ending Reddit exposes the current "page number" in the hash (thanks @larsjohnsen)
+- Reenable redditmedia/i.redd.it expandos (pending necessary changes on reddit's side in a few days) (thanks @andytuba)
```

Comment:
afaik yes. Additionally, it should work if you set reddit preference "media previews" to "expand preview based on subreddit settings" (default IIRC) or "automatically expand media previews".